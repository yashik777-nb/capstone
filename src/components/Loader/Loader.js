import React from "react";
import spinner from "./loader.gif";

export default class Loader extends React.Component {
  render() {
    return (
      <div>
        <img
          src={spinner}
          style={{
            width: "50px",
            margin: "auto",
            display: "block",
          }}
          alt="Loading..."
        />
      </div>
    );
  }
}
