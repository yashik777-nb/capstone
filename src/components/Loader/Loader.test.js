import React from "react";
import { configure } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { createMount } from "@material-ui/core/test-utils";

import Loader from "./Loader";

configure({ adapter: new Adapter() });

describe("Loader Tests", () => {
  let wrapper;

  beforeEach(() => {
    const render = createMount();
    wrapper = render(<Loader />);
  });

  it("Should render one image", () => {
    expect(wrapper.find("img").length).toEqual(1);
  });

  test("Should Render Loader page", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
