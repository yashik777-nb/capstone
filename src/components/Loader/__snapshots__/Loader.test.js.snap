// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Tests Should Render Loader page 1`] = `ReactWrapper {}`;
